docs_user/modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc

```diff
@@ -1,52 +1,27 @@
-[id="migrating-ceph-rbd_{context}"]
-
-//:context: migrating-ceph-rbd
-//kgilliga: This module might be converted to an assembly.
-
-= Migrating Ceph RBD
-
-In this scenario, assuming Ceph is already >= 5, either for HCI or dedicated
-Storage nodes, the daemons living in the OpenStack control plane should be
-moved/migrated into the existing external RHEL nodes (typically the compute
-nodes for an HCI environment or dedicated storage nodes in all the remaining
-use cases).
-
-== Requirements
-
-* Ceph is >= 5 and managed by cephadm/orchestrator.
-* Ceph NFS (ganesha) migrated from a https://bugzilla.redhat.com/show_bug.cgi?id=2044910[TripleO based deployment to cephadm].
-* Both the Ceph public and cluster networks are propagated, via TripleO, to the target nodes.
-* Ceph Mons need to keep their IPs (to avoid cold migration).
-
-== Scenario: Migrate mon and mgr from controller nodes
+[id="migrating-mon-and-mgr-from-controller-nodes_{context}"]
 
+= Migrating Ceph Monitor and Ceph Manager daemons to Red Hat Ceph Storage nodes
+//kgilliga: I'm trying to understand the purpose of this procedure. Is this procedure a prescriptive way for customers to migrate Ceph Monitor and Ceph manager daemons from controller nodes to Red Hat Ceph Storage nodes? Or are we recommending that customers create a proof of concept before doing the actual migration? And are oc0-controller-1 and oc0-ceph-0 just examples of the names of nodes for the purposes of this procedure?
```

Comment:
they are just examples. this document was a PoC but we might want to revisit it and turn into a procedure. At the time of writing EDPM wasn't a thing, so we might want to revisit the TripleO part of it.
I'll follow up in the next few weeks.